v2/checks/sbom/check_test.go

```diff
@@ -11,14 +11,12 @@ import (
 )
 
 func TestHasSBOM(t *testing.T) {
-	// TODO:CS come back and actually make the mocks for these, don't merge this in! It's making network requests!
-	mockService := sbomgcr.NewGCRService()
+	mockService := sbomgcr.NewMockGCRService("sha256-551182244aa6ab6997900bc04dd4e170ef13455c068360e93fc7b149eb2bc45f.att", "../../sbomgcr/fixtures/clouddo-sbom-oci")
```

Comment:
@thepwagner is there a more elegant way of passing in the fixture here? Not sure if this is the most idiomatic way to get the directory of the fixture to the gcr service.

Reply:
It LGTM - since the working directory is a promise made by the test framework, I think we can just rely on this relative path.

A couple of wild ideas:
* You could use "go embed", https://pkg.go.dev/embed , to embed the fixtures within the test binaries. Meh.
* You could try to find the module root, and build the path up from that, e.g. `"${root}/sbomgcr/fixtures/clouddo-sbom-oci"`. I think what you have is simpler and better than this.

Reply:
Also ideally I would like to use an sbom that is not shopify-specific